haas/cli.py

```diff
@@ -235,9 +169,11 @@ def check_status_code(response):
     else:
         sys.stdout.write(response.text + "\n")
 
-
+# Function object_url should be DELETED.
 # TODO: This function's name is no longer very accurate.  As soon as it is
 # safe, we should change it to something more generic.
+
+
 def object_url(*args):
     # Prefer an environmental variable for getting the endpoint if available.
     url = os.environ.get('HAAS_ENDPOINT')
```

Comment:
There are blank lines between documentation and functions that we should get rid of. 

About the `object_url` and other functions, I think we should move them to the end of `cli.py` and specify that they will be soon replaced by xxx. It is just a suggestion, what you you think @SahilTikale ?